### ISSUE TITLE
Exception: Layer lambda_1 does not support masking, but was passed an input_mask: Elemwise{neq,no_inplace}.0

### ISSUE THREAD

**Issue by PengTedShen** [codekansas/keras-language-modeling#22]
today I am trying insurance_qa_eval.py, got exception below :+1: 

  File "C:\Users\Deep_QA_System\data\insurance_qa_eval.py", line 261, in <module>
    model.compile(optimizer=optimizer)
  File "C:\Python35\lib\site-packages\keras\keras_models.py", line 111, in compile
    qa_model = self.get_qa_model()
  File "C:\Python35\lib\site-packages\keras\keras_models.py", line 96, in get_qa_model
    self._models = self.build()
  File "C:\Python35\lib\site-packages\keras\keras_models.py", line 247, in build
    question_pool = merge([maxpool(question_f_rnn), maxpool(question_b_rnn)], mode='concat', concat_axis=-1)
  File "C:\Python35\lib\site-packages\keras\engine\topology.py", line 485, in __call__
    self.add_inbound_node(inbound_layers, node_indices, tensor_indices)
  File "C:\Python35\lib\site-packages\keras\engine\topology.py", line 543, in add_inbound_node
    Node.create_node(self, inbound_layers, node_indices, tensor_indices)
  File "C:\Python35\lib\site-packages\keras\engine\topology.py", line 149, in create_node
    output_masks = to_list(outbound_layer.compute_mask(input_tensors[0], input_masks[0]))
  File "C:\Python35\lib\site-packages\keras\engine\topology.py", line 578, in compute_mask
    'but was passed an input_mask: ' + str(input_mask))
Exception: Layer lambda_1 does not support masking, but was passed an input_mask: Elemwise{neq,no_inplace}.0

Using Python 3.5 Keras-1.0.6

**Comment by codekansas** [comment]
This is addressed [here](https://github.com/codekansas/keras-language-modeling/issues/7)

**Comment by PengTedShen** [comment]
Thanks , it was solved.
